coveord/innovate-globe
# fix: globe metrics that update every minute [UA-8598]

## [UA-8598](https://coveord.atlassian.net/browse/UA-8598) :rocket:

### Proposed changes:

- Fix debugging message for when environment is set 🐛 
    - query.env was undefined in the production environment, ie. the url did not end with `?env=prd`, even though it does in a local deploy. This may be due to some [known](https://answers.netlify.com/t/querystringparameters-not-working-with-redirect-on-production/3828) netlify issues with redirection
    - Fixed by setting the environment to `prd` by default if the query param was undefined, and clarifying debugging logs
    - Even with no query param `?env=prd` at the end of the url, the default environment is now `prd`
- Removed extra check for an undefined environment in the useEffect that calls the `getMetrics()` method once per minute. 
    - `getMetrics()` already performs this check and reverts to a default value

### How to test

- Deployed locally ✅ 
- Deploy Preview works as expected ✅ 

[UA-8598]: https://coveord.atlassian.net/browse/UA-8598?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
